packages/contractkit/src/kit.ts

```diff
@@ -335,10 +344,6 @@ export class ContractKit {
       gasPrice: this.config.gasPrice,
     }
 
-    if (this.config.feeCurrency) {
```

Comment:
I think that this is helpful. Not everyone will set it as a default, but don't see why we should remove it.